### PR TITLE
✨ Add `--app-version` to the goreleaser packaging action

### PR DESCRIPTION
**Pull request by francostellari**
## Summary

Add `--app-version` to the goreleaser packaging action

## Related issue(s)

Fixes #

### CODE REVIEW

File: .github/workflows/goreleaser.yml [hunk 1]
```diff
@@ -61,6 +61,6 @@ jobs:
     - name: Package and push chart
       run: |
         make chart IMG=${{ env.REGISTRY }}/${{ env.OPERATOR_IMAGE }}:${{github.ref_name}}
-        helm package ${{ env.CHART_PATH }} --destination . --version ${{github.ref_name}}
-        helm push ./*.tgz oci://${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}    
+        helm package ${{ env.CHART_PATH }} --destination . --version ${{github.ref_name}} --app-version ${{github.ref_name}} --dependency-update
```
Comment (pdettori) [inline .github/workflows/goreleaser.yml:64]: @francostellari do we need the dependency update even if we do not use the sub-chart ?

**Comment by francostellari** [comment]
No but it does not hurt.
I thought to leave it in in case we need it in the future.
If you rather not risk it, I will remove it.

**Comment by pdettori** [comment]
We can leave it, it should not make a difference.

**Review by pdettori — APPROVED** [review]
/lgtm

**Pull request merged**